perseo_core-entrypoint.sh

```diff
@@ -30,6 +30,10 @@ touch /var/log/perseo/perseo-core.log
 
 ln -snf /dev/stdout /var/log/perseo/perseo-core.log &
 
+if [ "$PERSEO_LOG_LEVEL" ]; then
+    (sleep 10; curl -X PUT 'http://localhost:8080/perseo-core/admin/log?level='$PERSEO_LOG_LEVEL) &
```

Comment:
This type of technique may involve undesirable end-of-run conditions, could there be a better technique?

Reply:
Maybe you can provide one.

Reply:
@cesarjorgemartinez this PR provides a solution that suffices the implementetion of the env var and it has been  merged.

Of course, if you have a better solution, please do the corresponding PR so we can merge :)